docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,158 @@
+          // default value is 1.2. If the collection of documents being indexed have high occurrences
+          // of words that are not covered by a stop word filter, these words can quickly dominate any
+          // similarity calculation. In these cases, this value can be reduced to get more balanced results.
+          k1: 1.2,
+          // By default, we rank pages where the search term appears in the title higher than pages where
+          // the search term appears in just the text. This is done by "boosting" title matches with a
+          // higher value than content matches. The concrete boosting behavior can be controlled by changing
+          // the following settings.
+          titleBoost: 5,
+          contentBoost: 1,
+          parentCategoriesBoost: 2, // Only used when indexDocSidebarParentCategories > 0
+        },
+      },
+    ],
+  ],
+  presets: [
+    [
+      "classic",
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          id: "main",
+          routeBasePath: "/",
+          path: "docs",
+          sidebarPath: require.resolve("./sidebars.js"),
+          // Please change this to your repo.
+        },
+        blog: false,
+        theme: {
+          customCss: require.resolve("./src/css/custom.css"),
+        },
+      }),
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: require("./navigation.js"),
+      footer: {
+        style: "dark",
+        links: [
+          {
+            title: "Docs",
+            items: [
+              {
+                label: "Realm App Services Documentation",
+                to: "https://docs.mongodb.com/realm/",
+              },
+            ],
+          },
+          {
+            title: "Community",
+            items: [
+              {
+                label: "Stack Overflow",
+                href: "https://stackoverflow.com/questions/tagged/realm",
+              },
+              {
+                label: "Twitter",
+                href: "https://twitter.com/realm",
+              },
+              {
+                label: "MongoDB Developer Hub",
+                href: "https://www.mongodb.com/developer/"
+              }
+            ],
+          },
+          {
+            title: "More",
+            items: [
+              {
+                label: "GitHub",
+                href: "https://github.com/mongodb/docusaurus-realm",
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${new Date().getFullYear()} MongoDB, Inc. Built with Docusaurus.`,
+      },
+      prism: {
+        theme: lightCodeTheme,
+        darkTheme: darkCodeTheme,
+        // careful -- adding a nonexistent language will break
+        // the build without any error output!
+        additionalLanguages: ['kotlin', 'java'],
```

Comment:
```suggestion
        additionalLanguages: ['kotlin', 'java', 'dart', 'aspnet'],
```
Just added these in my version of the site, can confirm they work. And we want this syntax highlighting anyway.